### PR TITLE
Feature/users login

### PR DESCRIPTION
**Pull request by YourSouLi5Mine**
Acceptance criteria:
- Users can login
- Users can't access /me endpoint without being logged_in

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  def log_out
+    session.delete(:user_id)
+    @current_user = nil
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
```
Comment (ricvillagrana) [inline app/helpers/sessions_helper.rb]: Just a bit better:
```ruby
@current_user ||= User.find(session[:user_id])
```

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+    end
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (Egaribo) [inline app/helpers/sessions_helper.rb]: U can use !!current_user

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (YourSouLi5Mine) [reply app/helpers/sessions_helper.rb]: Read about it and find raises and error while find_by would return nil. Is it better to handle the exception?

File: app/helpers/sessions_helper.rb (hunk 2 above)
Comment (YourSouLi5Mine) [reply app/helpers/sessions_helper.rb]: Thanks, already fixed

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (YourSouLi5Mine) [reply app/helpers/sessions_helper.rb]: Fixed

**Pull request merged**